recipes-graphics/toyota/flutter-auto_2.0.bb

```diff
@@ -27,7 +27,7 @@ PLUGINS_COMMIT ??= "bbcf71a9e90e55ca4b63229b9be4e6ba995eb241"
 
 SRC_URI = "\
     gitsm://github.com/toyota-connected/ivi-homescreen.git;protocol=https;branch=v2.0;name=homescreen \
-    git://github.com/toyota-connected/ivi-homescreen-plugins.git;protocol=https;branch=v2.0;name=plugins;destsuffix=${S}/ivi-homescreen-plugins \
+    git://github.com/toyota-connected/ivi-homescreen-plugins.git;protocol=https;branch=v2.0;name=plugins;destsuffix=ivi-homescreen-plugins \
```

Comment:
not related to changes required for master, separate PR

Reply:
unfortunately it's necessary. in other case this package is unbuildable, because of wrong location of LICENSE and CMakeLists.txt files

Reply:
I think there need to be some cherry picks to master from scarthgap first

Reply:
you might not have noticed, but I started from scarthgap